Makefile

```diff
@@ -22,6 +22,14 @@ test: $(GOPATH)/bin/ginkgo
 		ginkgo -r --v --progress --trace --cover -- --v=3; \
 	fi
 
+test-existing: $(GOPATH)/bin/ginkgo
+	@if [ -z "${LINODE_API_TOKEN}" ]; then\
+		echo "Skipping Test, LINODE_API_TOKEN is not set";\
+	else \
```

Comment:
if we are using an existing cluster, we don't need an API token anymore, right?

Reply:
That is a good point ill remove that check.